src/components/common/GoodsItem/index.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import Image from '@components/common/Image';
+import Ranking from './Ranking';
+
+const IMAGE_SIZE_RANKING = 160;
+const IMAGE_SIZE_GOODS_ITEM = 250;
+const IMAGE_RADIUS = 4;
+
+export interface GoodsItemProps {
+  imageSrc: string;
+  subtitle: string;
+  title: string;
+  amount: number;
+  rankingIndex?: number;
+}
+
+export default function GoodsItem({ imageSrc, subtitle, title, amount, rankingIndex, ...rest }: GoodsItemProps) {
```

Comment:
```suggestion
export default function GoodsItem({ imageSrc, subtitle, title, amount, rankingIndex }: GoodsItemProps) {
```

전반적으로 rest operator를 자주 사용하시는거 같네요. 아래처럼 확장한 타입이 있고 이에 해당하는 props를 그대로 전달하고 싶을 때는 사용해도 좋지만 GoodsItemProps은 제한된 요소를 선언해서 사용하기에 지금은 불필요해보여요.

```tsx
interface PrimaryButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
  something: string;
}

export function PrimaryButton({ something, ...props }: PrimaryButtonProps) {
    // ...
    return <button {...props} />
}
```